RevenueCat/Scripts/Purchases.cs

```diff
@@ -88,7 +101,16 @@ private void Start()
     // Call this if you want to reset with a new user id
     private void Setup(string newUserId)
     {
-        _wrapper.Setup(gameObject.name, revenueCatAPIKey, newUserId, observerMode, userDefaultsSuiteName);
+        var apiKey = deprecatedLegacyRevenueCatAPIKey;
+        if (String.IsNullOrEmpty(apiKey))
+        {
+            if (Application.platform == RuntimePlatform.IPhonePlayer)
```

Comment:
does unity not support other apple platforms?

Reply:
We don't support MacOS at the moment. But I think the app could be built for MacOS using Catalyst and use the same API Key right?

Reply:
I believe Unity doesn't support Catalyst. 
@vegaro do you recall if there's a strong reason we don't support macOS? From our perspective, it shouldn't really matter if you're running macOS or not, right?

Reply:
happy to update this line to also include macOS, since it can't hurt

Reply:
I explored it in the past and we can't reuse our current iOS code. I actually started writing the code to support it and hit a wall that I don't remember so I reverted the changes. This was back when I was by myself and didn't have much Apple platforms experience. https://docs.unity3d.com/Manual/PluginsForDesktop.html